container/cli.py

```diff
@@ -107,6 +107,8 @@ def subcmd_build_parser(parser, subparser):
     subparser.add_argument('--services', action='store',
                            help=u'Rather than perform an orchestrated build, only build specific services.',
                            nargs='+', dest='service', default=None)
+    subparser.add_argument('--use-vault-password-file', action='store_true',
```

Comment:
I agree, I admit I took a shortcut while figuring out the relative path and the options. I'm new to python and I need to explore more the arguments parsing. I'll revisit to make it behave like the regular option. The password file would end up inside the builder container  via COPY (instead of volume mount in the current scenario). Any way to avoid files to be committed to the docker image, but just be loaded at runtime ?